src/snowflake/snowpark/modin/core/execution/dispatching/factories/baseio.py

```diff
@@ -279,66 +279,8 @@ def read_clipboard(cls, sep=r"\s+", **kwargs):  # pragma: no cover # noqa: PR01
         returns="""BaseQueryCompiler or dict/OrderedDict :
     QueryCompiler or OrderedDict/dict with read data.""",
     )
-    def read_excel(
-        cls,
-        io,
-        sheet_name=0,
-        header=0,
-        names=None,
-        index_col=None,
-        usecols=None,
-        squeeze=False,
-        dtype=None,
-        engine=None,
-        converters=None,
-        true_values=None,
-        false_values=None,
-        skiprows=None,
-        nrows=None,
-        na_values=None,
-        keep_default_na=True,
-        verbose=False,
-        parse_dates=False,
-        date_parser=None,
-        thousands=None,
-        comment=None,
-        skip_footer=0,
-        skipfooter=0,
-        convert_float=True,
-        mangle_dupe_cols=True,
-        na_filter=True,
-        **kwds,
-    ):  # noqa: PR01
-        if skip_footer != 0:
-            skipfooter = skip_footer
-        intermediate = pandas.read_excel(
-            io,
-            sheet_name=sheet_name,
-            header=header,
-            names=names,
-            index_col=index_col,
-            usecols=usecols,
-            squeeze=squeeze,
-            dtype=dtype,
-            engine=engine,
-            converters=converters,
-            true_values=true_values,
-            false_values=false_values,
-            skiprows=skiprows,
-            nrows=nrows,
-            na_values=na_values,
-            keep_default_na=keep_default_na,
-            verbose=verbose,
-            parse_dates=parse_dates,
-            date_parser=date_parser,
-            thousands=thousands,
-            comment=comment,
-            skipfooter=skipfooter,
-            convert_float=convert_float,
-            mangle_dupe_cols=mangle_dupe_cols,
-            na_filter=na_filter,
-            **kwds,
-        )
+    def read_excel(cls, **kwargs):  # noqa: PR01
```

Comment:
What was wrong with the old implementation here?

Reply:
Pandas would produce an error if it received an argument it didn't understand. I forget exactly which one it was here; but essentially there are two paths here:
1. Follow the read_csv approach and add all possible arguments for the current version of pandas ( and no more ). We need to do this for read_csv because we have both a server side and client side solution.
2. Just pass through kwargs into pandas and read the data frame.

In this case I went with (2) to keep the maintenance lighter; but I could be convinced of (1).